hail/python/hail/methods/impex.py

```diff
@@ -2972,6 +2972,325 @@ def import_avro(paths, *, key=None, intervals=None):
+    combined = combined.checkpoint(checkpoint_path, overwrite=True)
+
+    new_partitioner = combined.reference_data._calculate_new_partitions(combined.reference_data.n_partitions())
+
+    info('import_gvs: repartitioning and writing final VDS')
+
+    # add reference base to reference data table as a Hail VDS requirement. Add from FASTA file.
+    rd = add_reference_allele(
+        hl.read_matrix_table(os.path.join(checkpoint_path, 'reference_data'),
+                             _intervals=new_partitioner))
+
+    vd = hl.read_matrix_table(os.path.join(checkpoint_path, 'variant_data'),
+                              _intervals=new_partitioner)
+
+    # read site and vqsr data with same intervals for efficient joins
+    site = hl.read_table(site_path, _intervals=new_partitioner)
+    vqsr = hl.read_table(vqsr_path, _intervals=new_partitioner)
+
+    vd = vd.annotate_rows(filters=hl.coalesce(site[vd.locus].filters, hl.empty_set(hl.tstr)))
+
+    # vqsr ref/alt come in normalized individually, so need to renormalize to the dataset ref allele
+    vd = vd.annotate_rows(as_vqsr = hl.dict(vqsr.index(vd.locus, all_matches=True)
+                                            .map(lambda record: (record.alt + vd.alleles[0][hl.len(record.ref):], record.drop('ref', 'alt')))))
+    vd = vd.annotate_globals(tranche_data=tranche.collect(_localize=False),
+                             truth_sensitivity_snp_threshold=truth_sensitivity_snp_threshold,
+                             truth_sensitivity_indel_threshold=truth_sensitivity_indel_threshold)
+
+    sorted_tranche_data = hl.sorted(vd.tranche_data, key=lambda x: x.truth_sensitivity)
+    vd = vd.annotate_globals(snp_vqslod_threshold=
+                             sorted_tranche_data.filter(lambda x: (x.model == 'SNP') & (
+                                         x.truth_sensitivity >= truth_sensitivity_snp_threshold))
+                             .head()
+                             ,
+                             indel_vqslod_threshold=sorted_tranche_data.filter(lambda x: (x.model == 'INDEL') & (
+                                         x.truth_sensitivity >= truth_sensitivity_indel_threshold))
+                             .head()
+                             )
+
+    vd = vd.annotate_rows(
+        allele_NO=vd.alleles[1:].map(
+            lambda allele: hl.coalesce(vd.as_vqsr.get(allele).yng_status == 'N', False)),
+        allele_YES=vd.alleles[1:].map(
+            lambda allele: hl.coalesce(vd.as_vqsr.get(allele).yng_status == 'Y', True)),
+        allele_OK=vd.alleles[1:].map(
+            lambda alt: hl.coalesce(vd.as_vqsr.get(alt).vqslod > hl.if_else(hl.is_snp(vd.alleles[0], alt),
+                                                                            truth_sensitivity_snp_threshold,
+                                                                            truth_sensitivity_indel_threshold),
+                                    True)),
```

Comment:
we want to be comparing vd.as_vqsr.get(alt).vqslod and snp_vqslod_threshold.min_vqslod which we just created with the method above.

Reply:
(and I did not test it out, but presumably the same for the indels)